granule.py

```diff
@@ -0,0 +1,391 @@
+"""

```

Comment:
Oh I just noticed this file is here twice. I made the suggestions on this one, but actually the second version is the one that should be kept and this removed.